build.sh

```diff
@@ -34,13 +65,43 @@ done
 
 SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-# Copy Thunder prject to Dockerfile context if project path is provided
-if [[ "${PROJECT_PATH}" != "" ]]; then
-    cp --dereference --recursive "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+# Copy Thunder project to Dockerfile context if project path is provided
+if [ "${PROJECT_PATH}" != "" ]; then
+  if [ "${OS_NAME}" == "osx" ]; then
+      rm -rf "${SCRIPT_DIRECTORY}/www"
+      cp -R "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+  else
+      rm --recursive --force  "${SCRIPT_DIRECTORY}/www"
+      cp --dereference --recursive "${PROJECT_PATH}" "${SCRIPT_DIRECTORY}/www"
+  fi
```

Comment:
As discussed, let's simplify all if/else's by just using shorthand syntax, which runs on every OS

Reply:
I've done this in the latest commit. The one thing we can't have as far as I can see is the --deference option. Doesn't exist on OSX.

Reply:
Nothing broke so yay for less complexity.